diff --git a/build/config/android/rules.gni b/build/config/android/rules.gni
index 784e39ee221e0014967b90ae6cb8dea5112afcf9..265603a85f340651a68518094d301767d44f0b61 100644
--- a/build/config/android/rules.gni
+++ b/build/config/android/rules.gni
@@ -647,6 +647,7 @@ if (enable_java_templates) {
   #     output = "$target_gen_dir/AndroidManifest.xml"
   #   }
   template("jinja_template") {
+    if (defined(invoker.includes)) { invoker.includes += brave_android_manifest_includes } else { invoker.includes = brave_android_manifest_includes }
     action_with_pydeps(target_name) {
       # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
       run_under_python2 = true
