diff --git a/chrome/browser/resources/settings/BUILD.gn b/chrome/browser/resources/settings/BUILD.gn
index 13445757b2bebcbcec9da49d9ccbd345b524510e..5e4e773d07041597d1d115d54e68adaa07c2a33b 100644
--- a/chrome/browser/resources/settings/BUILD.gn
+++ b/chrome/browser/resources/settings/BUILD.gn
@@ -93,6 +93,7 @@ generate_grd("build_grd") {
       "$target_gen_dir/$preprocess_gen_manifest",
     ]
   }
+  deps += [ "//brave/browser/resources/settings:resources" ]
 }
 
 preprocess_if_expr("preprocess") {
